internal/pkg/auth/storage.go

```diff
@@ -260,3 +260,16 @@ func createEncodedTextFile() error {
 
 	return nil
 }
+
+func GetPrettyAuthFlow(flow AuthFlow) string {
```

Comment:
do we really need this if it's just used in debug logs? :thinking:

Reply:
Yes, maybe we don't. I'll just print the raw string, its very readable